README.md

```diff
@@ -25,41 +25,40 @@ License URL	: http://forgerock.org/cddlv1-0/
+### Downloading the project code and loading it into an IDE such as IntelliJ
 
 You need to get the code locally for the git project repo. For this, you need to
 
-- Fork the project https://stash.forgerock.org/projects/OB/repos/openbanking-sample/browse
-- Clone your fork (if it's your first development with GIT, we recommend putting all the git repo in ~/Development/GIT)
-- Download the latest intellij version (ultimate)
-- Open the project with intellij (you will need to open it as an existing maven project)
-- Install mongod if you don't have it already. There is plenty of doc online for this. We recommend using brew
-- You will need later to run mongod in background. Either do it in console before working or setup an external tool 
-in intellij so you can run it from intellij (tips: https://stackoverflow.com/questions/33595827/is-there-a-way-of-running-mongod-from-webstorm-mac)
+- Fork the project from the following URL: https://github.com/ForgeRock/ForgeRock-OpenBanking-Sample
+- Clone your fork
+- Open the project in your IDE
+- Install MongoDB. If you're installing on a Mac, we recommend using `brew`
+- Prepare to run MongoDB in the background. You can do so from either the console or with an external tool associated with your IDE.
+(For one method: https://stackoverflow.com/questions/33595827/is-there-a-way-of-running-mongod-from-webstorm-mac)
 
 
-### Setup the host files
+### Set up host files
 
-You will need to create some new hostnames for the application.
+If you're running this sample locally, associated the following hostnames with the localhost IP address:
 
 ```$xslt
 127.0.0.1		tpp.sample.ob.forgerock.financial redirect.tpp.sample.ob.forgerock.financial tpp.ob.forgerock.financial
 ```
 
 ### Register your TPP with the ForgeRock directory
 
-ForgeRock directory manage keys for you and expose the jwkms, a service for generated JWTs.
+The ForgeRock directory manages keys for you and exposes the jwkms, a service that manages generated JWTs.
 
-What you need to do:
-- create a new software statement into ForgeRock directory
-- download your transport key
-- Setup postman to use your transport keys for the jwkms, directory and ASPSP
-- run the collections ASPSP V1.1 > Examples > Example 1: Onboarding
+To register your TPP:
+- Create a new software statement in the ForgeRock directory.
+- Download your transport key.
+- Set up an API Development Environment to use your transport keys for the jwkms, directory and ASPSP. (We use Postman.)
+- In the API Development Environment, run the following collection: ASPSP V1.1 > Examples > Example 1: Onboarding.
 
 ### Register your TPP with the Open Banking directory
 
-We implemented a script which will allow you to generate keys, ready to go, that you can import to Open Banking directory.
-What you need to do:
+We've included a script which will allow you to generate keys that you can import to Open Banking directory.
+To proceed:
```

Comment:
I think this section needs additional clarification, but I haven't taken the time to analyze the steps in this section.